src/Stratis.Bitcoin/P2P/PeerAddress.cs

```diff
@@ -80,23 +80,58 @@ public IPAddress Loopback
         public DateTime? LastSeen { get; private set; }
 
         /// <summary>
-        /// The UTC Date when a Peer is banned.
-        /// <para>
-        /// This is set in <see cref="PeerBanning"/>, against <see cref="PeerAddress"/>.
-        /// </para>
+        /// UTC DateTime when a peer is banned.
```

Comment:
UTC timestamp when ...
or
Time in UTC when ...

Otherwise you would need to use `<see>` if you wanted to reference the DateTime class